### PR TITLE
Adding a short description of the error to a test comment

### PR DESCRIPTION
**Pull request by Nypiaka**
adding a short description of the error to a test comment

### CODE REVIEW

File: server/src/printers/TestsPrinter.cpp [hunk 1]
```diff
@@ -408,6 +408,9 @@ void TestsPrinter::genHeaders(Tests &tests, const fs::path& generatedHeaderPath)
 }
 
 void TestsPrinter::testHeader(const Tests::MethodTestCase &testCase) {
+    if (testCase.isError()) {
+        ss << "//" << testCase.getError() << NL;
```
Comment (ladisgin) [inline server/src/printers/TestsPrinter.cpp:412]: use method strComment

File: server/src/Tests.h [hunk 2]
```diff
@@ -471,6 +471,11 @@ namespace tests {
             FileInfo getFileByName(char fileName) const {
                 return filesValues.value()[fileName - 'A'];
             }
+            
+            std::string getError() const {
+                assert(!errorDescriptors.empty());
```
Comment (ladisgin) [inline server/src/Tests.h:476]: Just return default value instead of crash program

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/src/printers/TestsPrinter.cpp (hunk 1 above)
Comment (Nypiaka) [reply server/src/printers/TestsPrinter.cpp]: changed

File: server/src/Tests.h (hunk 2 above)
Comment (Nypiaka) [reply server/src/Tests.h]: changed

**Review by ladisgin — APPROVED** [review]

**Pull request merged**